libgnucash/app-utils/config-var.scm

```diff
@@ -33,6 +33,11 @@
 ;;; permanent, and if they leave the variable value different from the
 ;;; default, should be saved to the auto configuration file.
 
+;; this module should be deprecated.
+;; (issue-deprecation-warning "config-var.scm must be deprecated.")
+;; dummy line above to be searchable in master
+;; its defined functions are no longer used in code.
+
```

Comment:
Hmm, I don't understand this part: why can't we add the (issue-deprecation-warning ...) message to each of the individual gnc:config-var-* functions ?
Is this because just compiling that code would already trigger the deprecation notices ?

Reply:
No, it wouldn't trigger... it's fine to do individually. Except I don't understand this module which is unused apart from a comment by linas in 1999 in prefs.scm:20 -- I wouldn't know alternative either since this module is completely unused.